tests/mocks/libs/paseto.ts

```diff
@@ -0,0 +1,5 @@
+export const makePasetoMock = () => ({
+	sign: jest.fn(),
+    bytesToKeyObject: jest.fn(),
```

Comment:
Be sure that they have the same identation (The recommended extensions can help with that)